### ISSUE TITLE
[Feature]: Make image metadata available from image widget

### ISSUE THREAD

**Issue by AS-Laguna** [appsmithorg/appsmith#17076]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

As a user I should be able to access the properties of an image such as it's width and height. We should add a new property to the image widget like `Image.metadata` that contains the data of the current image.

### Why should this be worked on?

A user needs to send the image height and width to their production app after uploading the image to Appsmith. Their app uses these properties to optimize before rendering.

These properties can be used for performing logic based on image data. This could be used to filter images by size or date. You could also use this to limit uploading images to an API based on it's dimensions, date, or size.

**Comment by mjpsr11** [comment]
Since 'document' is not available in a JS object, this information is unavailable. Normally, image properties can be retrieved simply from the dom after creating an img element.

The Image widget already renders an image from a url, so properties should be readily available.

This feature is greatly needed to optimize performance for image intensive apps. A good example is Next JS and the Image tag using fill (outer element must be sized) or fixed (inner element must be sized) properties.

**Comment by Svarto** [comment]
any update on this, I also need this feature...

**Comment by dilippitchika** [comment]
@Svarto no, we haven't prioritised this on our end, but with JS libraries i think you can import this library - https://github.com/exif-js/exif-js now to understand the image metadata

**Comment by Svarto** [comment]
@dilippitchika that would work great...not sure how to import this library though? I don't have it in my dependency list and docs don't mention that exif-js is included?
